Octokit.Tests.Integration/Clients/PullRequestsReviewRequestClientTests.cs

```diff
@@ -0,0 +1,164 @@
+
+        _client = _github.PullRequest.ReviewRequest;
+
+        _context = _github.CreateRepositoryContext("test-repo").Result;
+        _github.Repository.Collaborator.Add(Helper.UserName, _context.RepositoryName, collaboratorLogin);
+    }
+
+    [IntegrationTest]
+    public async Task CanGetAllWhenNone()
+    {
+        var pullRequest = await CreatePullRequest(_context);
+
+        var reviewRequests = await _client.GetAll(Helper.UserName, _context.RepositoryName, pullRequest.Number);
+
+        Assert.NotNull(reviewRequests);
+        Assert.Empty(reviewRequests);
+    }
+
+    [IntegrationTest]
+    public async Task CanCreateAndThenGetAll()
+    {
+        var pullRequest = await CreatePullRequest(_context);
+        var reviewers = new List<string> { collaboratorLogin };
+        var reviewRequestToCreate = new PullRequestReviewRequest(reviewers);
+
+        await _client.Create(Helper.UserName, _context.RepositoryName, pullRequest.Number, reviewRequestToCreate);
+        var reviewRequests = await _client.GetAll(Helper.UserName, _context.RepositoryName, pullRequest.Number);
+
+        Assert.Equal(reviewers, reviewRequests.Select(rr => rr.Login));
+    }
+
+    [IntegrationTest]
+    public async Task CanCreate()
+    {
+        var pullRequest = await CreatePullRequest(_context);
+        var reviewers = new List<string> { collaboratorLogin };
+        var reviewRequestToCreate = new PullRequestReviewRequest(reviewers);
+
+        var pr = await _client.Create(Helper.UserName, _context.RepositoryName, pullRequest.Number, reviewRequestToCreate);
+
+        Assert.Equal(reviewers, pr.RequestedReviewers.Select(rr => rr.Login));
+    }
+
+    [IntegrationTest]
+    public async Task CanCreateAndDelete()
+    {
+        var pullRequest = await CreatePullRequest(_context);
+        var reviewers = new List<string> { collaboratorLogin };
+        var reviewRequestToCreate = new PullRequestReviewRequest(reviewers);
+
+        await _client.Create(Helper.UserName, _context.RepositoryName, pullRequest.Number, reviewRequestToCreate);
+        var reviewRequestsAfterCreate = await _client.GetAll(Helper.UserName, _context.RepositoryName, pullRequest.Number);
+        await _client.Delete(Helper.UserName, _context.RepositoryName, pullRequest.Number, reviewRequestToCreate);
+        var reviewRequestsAfterDelete = await _client.GetAll(Helper.UserName, _context.RepositoryName, pullRequest.Number);
+
+        Assert.Equal(reviewers, reviewRequestsAfterCreate.Select(rr => rr.Login));
+        Assert.Empty(reviewRequestsAfterDelete);
+    }
+
+    public void Dispose()
+    {
+        _context.Dispose();
+    }
+
+    /// <summary>
+    /// Creates the base state for testing (creates a repo, a commit in master, a branch, a commit in the branch and a pull request)
+    /// </summary>
+    /// <returns></returns>
+    async Task<PullRequestData> CreatePullRequest(RepositoryContext context, string branch = branchName)
```

Comment:
For consistency it would be nice to create the base state using the same process/code that the `CreateTheWorld()` functions in the `PullRequestsClientTests` use:   https://github.com/octokit/octokit.net/blob/master/Octokit.Tests.Integration/Clients/PullRequestsClientTests.cs#L931-L993

That way we have consistency in how we create Pull Requests for integration tests